Cargo.toml

```diff
@@ -19,7 +19,7 @@ description = "An n-dimensional array for general elements and for numerics. Lig
 keywords = ["array", "data-structure", "multidimensional", "matrix", "blas"]
 categories = ["data-structures", "science"]
 
-exclude = ["docgen/images/*"]
+include = ["src", "LICENSE-MIT", "LICENSE-APACHE", "RELEASES.md", "README.rst", "README-quick-start.md"]
```

Comment:
I read the documentation and I think what you wrote is ok. However:

-  I wonder about `ndarray-rand`. Shouldn't it be included?
- I'd prefer using `/src` than `src`.

Reply:
You're right, `ndarray-rand` is another crate. Thank you for your contribution.